### PR TITLE
image article

### PR DESCRIPTION
**Pull request by IslandRhythms**

### CODE REVIEW

File: tutorials/vue/img.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+To use the `img` tag in vue, you must use the `v-bind:src` directive or `:src` for short.
+Be sure to enclose the image address in single quotes on top of the double quotes.
```
Comment (vkarpov15) [inline tutorials/vue/img.md:2]: Replace with "Remember that `:src` expects a JavaScript expression, so if you want to use a string in `:src` you need to wrap the string in quotes."

File: tutorials/vue/img.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+```javascript
+  const app = new Vue({
+    template: `
+      <div style="width: 50%">
+       <img :src="'../../assets/logo.png'" />
```
Comment (vkarpov15) [inline tutorials/vue/img.md:8]: Also add an example of using `:src` with a variable in `data`

File: tutorials/vue/img.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      </div>
+    `,
+  });
+  app.$mount("#content");
+```
+
+<div id="content"></div>
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12"></script>
```
Comment (vkarpov15) [inline tutorials/vue/img.md:16]: Use unpkg instead like in our [Vue CDN](https://masteringjs.io/tutorials/vue/cdn) tutorial

File: tutorials/vue/img.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+       <img :src="'../../assets/logo.png'" />
+      </div>
+    `,
+  });
+  app.$mount("#content");
+</script>
+
+## Computed src
+
+You can also pass a computed property to `:src`, but make sure that it returns the image file location.
```
Comment (vkarpov15) [inline tutorials/vue/img.md:30]: What does "image file location" mean here?

File: tutorials/vue/img.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+
+```javascript
+<div id="computed"></div>
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12"></script>
```
Comment (vkarpov15) [inline tutorials/vue/img.md:35]: Use unpkg instead

File: tutorials/vue/img.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+        return '../../assets/logo.png';
+      }
+    }
+  });
+  example.$mount("#computed");
+</script>
+```
+
+<div id="computed"></div>
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12"></script>
```
Comment (vkarpov15) [inline tutorials/vue/img.md:54]: Unpkg

File: tutorials/vue/img.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+      getPhoto() {
+        return '../../assets/logo.png';
+      },
+      getStyle() {
+        return {active: this.active}
+      }
+    },
+    mounted() {
+      this.active = true;
+      this.$nextTick().then(() => {
```
Comment (vkarpov15) [inline tutorials/vue/img.md:100]: Is this necessary?

File: tutorials/vue/img.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+      <div style="width: 50%">
+       <img ref="result" :class="getStyle" :src="getPhoto" />
+       <div> classes attached to this image are: {{result}}</div>
+      </div>
+    `,
+    computed: {
+      getPhoto() {
+        return '../../assets/logo.png';
+      },
+      getStyle() {
```
Comment (vkarpov15) [inline tutorials/vue/img.md:94]: Rename this to `imageClass`. It is a bit strange to name a computed property `getX`

**Review by vkarpov15 — CHANGES_REQUESTED** [review]

File: tutorials/vue/img.md (hunk 7 above)
Comment (IslandRhythms) [reply tutorials/vue/img.md:100]: Yes, otherwise the classes won't load in time.

File: tutorials/vue/img.md (hunk 4 above)
Comment (IslandRhythms) [reply tutorials/vue/img.md:30]: I'll change it to "path to the image file"

**Pull request merged**